apps/condo/domains/common/components/containers/amplitude/AmplitudeAuthorizedUser.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { Amplitude, useAmplitude } from 'react-amplitude-hooks'
+import { useAuth } from '@core/next/auth'
+import { useOrganization } from '@core/next/organization'
+import omit from 'lodash/omit'
+import { BaseEventProperties } from '@condo/domains/common/components/containers/amplitude/AmplitudeProvider'
+import { OrganizationEmployee } from '@condo/domains/organization/utils/clientSchema'
+
+const USER_OMITTED_FIELDS = ['phone', 'email', '__typename', 'avatar', 'isAdmin']
+
+export type AmplitudeUserProperties = Partial<BaseEventProperties['user']> & {
+    organization?: string
+    role?: string
+}
+
+export type AuthorizedUserEventProperties = Pick<BaseEventProperties, 'page'> & AmplitudeUserProperties
+
+const AmplitudeAuthorizedUser: React.FC = ({ children }) => {
+    const { user } = useAuth()
+    const { organization } = useOrganization()
+    const { eventProperties } = useAmplitude()
+
+    let userProperties: AmplitudeUserProperties = {}
+    let organizationEmployeeWhere = {}
+
+    if (user && organization) {
+        organizationEmployeeWhere = {
+            user: {
+                id: user.id,
+            },
+            organization: {
+                id: organization.id,
+            },
+            isAccepted: true,
+        }
+    }
+
+    const { obj: userOrganization } = OrganizationEmployee.useObject(
```

Comment:
why you don't use a `link` from `useOrganisation`? why you need to another one request

Reply:
Fixed this